user_tools/src/spark_rapids_pytools/rapids/dev/instance_description.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Implementation class representing wrapper around the RAPIDS acceleration Prediction tool."""
+
+import os
+from dataclasses import dataclass
+
+from spark_rapids_pytools.rapids.rapids_tool import RapidsTool
+from spark_rapids_pytools.common.sys_storage import FSUtil
+from spark_rapids_pytools.common.utilities import Utils
+from spark_rapids_pytools.cloud_api.sp_types import get_platform
+from spark_rapids_pytools.rapids.tool_ctxt import ToolContext
+
+
+@dataclass
+class InstanceDescription(RapidsTool):
+    """Wrapper layer around Generate_Instance_Description Tool."""
+
+    name = 'instance_description'
+
+    def _connect_to_execution_cluster(self):
```

Comment:
For methods with not return, set their return to None
```
_connect_to_execution_cluster(self) -> None:
```

Reply:
Thanks @amahussein! Fixed all review feedback.